tests/console/x_vt.pm

```diff
@@ -15,11 +15,11 @@ use base "consoletest";
 use strict;
 use warnings;
 use testapi;
-use version_utils 'is_tumbleweed';
+use version_utils qw(is_tumbleweed is_leap);
 
 sub run {
     if (script_run("ps -ef | grep bin/X | egrep 'tty7|wayland'") == 1) {
-        if (is_tumbleweed && script_run('ps -ef | grep bin/X | grep tty2') == 0) {
+        if ((is_tumbleweed || is_leap('>=15.2')) && script_run('ps -ef | grep bin/X | grep tty2') == 0) {
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check. This was already done wrong in the beginning.